doc/source/known-issues.rst

```diff
@@ -1,97 +1,126 @@
+.. index:: Known Issues
+.. _Known Issues:
+
 Known Issues
 ============
 
-This is a short list of known issues.
+This section provides a short list of known issues.
+
+.. index:: Known Issues, Mount Path Limit
+.. _Mount Path Limit:
 
-88 character mount path limitation
-----------------------------------
+Mount Path Limit
+----------------
 
-There is a know mountpoint path length limitation issue on FreeBSD which is set to a historical 88 character limit.
+There is a known mountpoint path length limitation issue on FreeBSD.
+Path length has an historical 88 character limit.
 
-This issue does not affect iocage jails from functioning properly, but can present challenges
-when diving into ZFS snapshots (cd into .zfs/snapshots, tar, etc.).
+This issue does not affect :command:`iocage` jails from functioning
+properly, but can present challenges when diving into ZFS snapshots,
+like :command:`cd` into :file:`.zfs/snapshots`, :command:`tar`, etc.
 
 ZFS snapshot creation and rollback is not affected.
 
-To workaround this issue iocage 1.6.0 introduced a ``hack88`` property.
+To work around this issue, **iocage 1.6.0** introduced a **hack88**
+property.
 
 Example:
 
 Shut down jail:
 
-``iocage stop myjail``
+:samp:`# iocage stop myjail`
 
-Set the ``hack88`` property to "1":
+Set the **hack88** property to *1*:
 
-``iocage set hack88=1``
+:samp:`# iocage set hack88=1 myjail`
 
 Start jail:
 
-``iocage start myjail``
+:samp:`# iocage start myjail`
 
-To revert back to full paths repeat the procedure but set ``hack88=0``.
+Revert back to full paths by repeating the procedure, but setting
+**hack88=0**.
 
-To create a system wide default (introduced in 1.6.0) for all newly created jails use:
+Create a system wide default (introduced in **1.6.0**) for all newly
```

Comment:
This is also no longer needed, you cannot set short for all jails by default, it needs to be specified. Also removing old version numbers.